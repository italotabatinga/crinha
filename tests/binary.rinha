print (3+5) // Add
print ("a" + 2)
print (2 + "a")
print ("a" + "b")
print (0 - 1) // Sub
print (2 * 2) // Mul
print (3 / 2) // Div
print (4 % 2) // Rem
print ("a" == "a") // Eq
print (2 == 1 + 1)
print (true == true)
print (false == false)
print (true == false)
print ("a" != "b") // Neq
print ("a" != "a")
print (3 != 1 + 1)
print (true != false)
print (true != true)
print (1 < 2) // Lt
print (2 < 2)
print (3 < 2)
print (1 > 2) // Gt
print (2 > 2)
print (3 > 2)
print (1 <= 2) // Lte
print (2 <= 2)
print (3 <= 2)
print (1 >= 2) // Gte
print (2 >= 2)
print (3 >= 2)
print (true && false) // And
print (false || true) // Or