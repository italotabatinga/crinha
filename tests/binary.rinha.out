8
a2
2a
ab
-1
4
1
0
true
true
true
true
false
true
false
true
true
false
true
false
false
false
false
true
true
true
false
false
true
true
false
true
